src/main/java/uk/gov/digital/ho/hocs/casework/security/UserPermissionsService.java

```diff
@@ -1,25 +1,26 @@
 package uk.gov.digital.ho.hocs.casework.security;
 
-import lombok.extern.slf4j.Slf4j;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import uk.gov.digital.ho.hocs.casework.application.RequestData;
 import uk.gov.digital.ho.hocs.casework.client.infoclient.InfoClient;
 import uk.gov.digital.ho.hocs.casework.client.infoclient.PermissionDto;
 import uk.gov.digital.ho.hocs.casework.client.infoclient.TeamDto;
 
 import java.nio.BufferUnderflowException;
-import java.util.*;
+import java.util.Comparator;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 @Service
-@Slf4j
 public class UserPermissionsService {
 
-    private RequestData requestData;
-    private InfoClient infoClient;
+    private final RequestData requestData;
```

Comment:
👮 I'm glad the final police have swung by. this is the most important change in this PR IMO

Reply:
Nee noor nee noor 👮🏽